src/CommitmentsV2/SFA.DAS.CommitmentsV2/Validators/DraftApprenticeshipDetailsValidator.cs

```diff
@@ -0,0 +1,159 @@
+﻿using FluentValidation;
+using FluentValidation.Validators;
+using SFA.DAS.CommitmentsV2.Domain.Entities;
+using SFA.DAS.CommitmentsV2.Domain.Extensions;
+using SFA.DAS.CommitmentsV2.Domain.Interfaces;
+using SFA.DAS.CommitmentsV2.Domain.ValueObjects;
+using TrainingProgrammeStatus = SFA.DAS.Apprenticeships.Api.Types.TrainingProgrammeStatus;
+
+namespace SFA.DAS.CommitmentsV2.Validators
+{
+    public class DraftApprenticeshipDetailsValidator : AbstractValidator<DraftApprenticeshipDetails>
+    {
+        private class StartDateValidator : AbstractValidator<DraftApprenticeshipDetails>
+        {
+            public StartDateValidator(IAcademicYearDateProvider academicYearDateProvider)
+            {
+                RuleFor(ctx => ctx.StartDate)
+                    .GreaterThanOrEqualTo(Constants.DasStartDate)
+                    .WithMessage($"The start date must not be earlier than {Constants.DasStartDate:MM yyyy}");
+
+                RuleFor(ctx => ctx.StartDate)
```

Comment:
If the user falls foul of this rule and the one above, then two errors surface in the UI. This is not the desired behaviour; they should see only one. There was a unit test covering that scenario, but the expected result is now being discarded so the assert doesn't surface the failure.